services/src/test/java/io/scalecube/services/DummyRouter.java

```diff
@@ -0,0 +1,23 @@
+package io.scalecube.services;
+
+import static com.google.common.base.Preconditions.checkArgument;
+
+import io.scalecube.services.routing.Router;
+import io.scalecube.transport.Message;
+
+import java.util.Optional;
+
+public class DummyRouter implements Router {
+  private Object empty;
+
+  public DummyRouter() {
+    checkArgument(this.empty != null);
+  }
+
+  @Override
+  public Optional<ServiceInstance> route(Message request) {
+    // TODO Auto-generated method stub
+    return null;
```

Comment:
Basically if `null` is supposed behavior then better to return empty optional instead and remove TODO comment above...